EdenGri/sonic-swss-common
# Persistent loglevel

This PR needs to be merged only after the following PR: https://github.com/dprital/sonic-buildimage/pull/12
This PR needs to be merged with this PR: https://github.com/EdenGri/sonic-utilities/pull/7

What I did
Add the ability to the user to save the loglevel and make it persistent to reboot.

How I did it
Move the logger tables from the LOGLEVEL DB to the CONFIG DB

How to verify it
1. change the orchagent loglevel (for example) -> swssloglevel -c orchagent -l DEBUG
2. save the loglevel -> run config save
3. reboot
4. verify that the orchagent log level is still DEBUG ->run run redis-cli -n 4 hgetall "LOGGER|orchagent"
